addon/components/roadsigns-table.hbs

```diff
@@ -53,5 +53,25 @@
         {{/if}}
       {{/if}}
     {{/if}}
+    <div class="au-c-pagination au-u-margin-top">
```

Comment:
probably cleanest to move this into its own component

Reply:
I would also hide pagination in case there are no results, currently shows "0-9" of <blank> if there are no results